test/character_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+
+class CharacterTest < Minitest::Test
+
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+  end
+
+  def test_it_exists
+    assert_equal Character, @kitt.class
```

Comment:
generally we'll use `assert_instance_of Character, @kitt` here instead of asserting a `.class` comparison